action.yml

```diff
@@ -16,6 +16,10 @@ inputs:
     required: false
     default: 'gh-pages'
   
+  destinationDir:
+    description: 'The directory to push the built site to'
```

Comment:
Change the description to: 
```suggestion
    description: 'The directory in the repo to place the site in'
```
And move it to after the `repo` option (I should really move the `branch` option as well).